File: app/src/main/java/com/mailerdaemon/app/events/EventModel.java
package com.mailerdaemon.app.events;

import com.google.firebase.firestore.ServerTimestamp;

import java.util.Date;
import java.util.List;

import lombok.Data;

@Data
class EventModel {
  List<PostModel> posts;
  private String name;
  @ServerTimestamp
  private Date date;
}

File: app/src/main/java/com/mailerdaemon/app/events/EventModel.kt
package com.mailerdaemon.app.events

import com.google.firebase.firestore.ServerTimestamp
import java.util.Date

data class EventModel(
    var posts: List<PostModel>? = null,
    var name: String? = null,
    @ServerTimestamp
    var date: Date? = null
)
